internals/daemon/api_checks.go

```diff
@@ -66,3 +67,34 @@ func v1GetChecks(c *Command, r *http.Request, _ *UserState) Response {
 	}
 	return SyncResponse(infos)
 }
+
+func v1PostChecks(c *Command, r *http.Request, _ *UserState) Response {
+	var payload struct {
+		Action string `json:"action"`
+		Check  string `json:"check"`
+	}
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&payload); err != nil {
+		return BadRequest("cannot decode data from request body: %v", err)
+	}
+	switch payload.Action {
+	case "run":
+		if payload.Check == "" {
+			return BadRequest("no check to %s provided", payload.Action)
```

Comment:
Let's again be consistent in our error messages with other code. Looks like it's usually like "must specify check name"

Reply:
"must specify" does seem more common here, but services has it like the above:

```
		if len(payload.Services) == 0 {
			return BadRequest("no services to %s provided", payload.Action)
		}
```

Reply:
Hmm, yeah, we should pick one. Looks like "must specify" is significantly more common:

```
api_exec.go:		return BadRequest("must specify command")
api_files.go:			return BadRequest("must specify one or more paths")
api_files.go:			return BadRequest(`must accept multipart/form-data`)
api_files.go:			return BadRequest("must specify path")
api_files.go:		return BadRequest("must specify one or more files")
api_services.go:			return BadRequest("no services to %s provided", payload.Action)
api_signals.go:		return BadRequest("must specify one or more services")
```

In addition, "no services to start provided" reads awkwardly to me -- "provided" seems like it's in the wrong place. Let's change that one to `must specify services for %s action`.